google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -2442,4 +2443,69 @@ public void testGetServiceAccount() {
     assertNotNull(serviceAccount);
     assertTrue(serviceAccount.getEmail().endsWith(SERVICE_ACCOUNT_EMAIL_SUFFIX));
   }
+
+  @Test
```

Comment:
Add an integration test to verify ACL and DefaultObjectACL didn't change when enabling BPO.
1. Save ACL field values.
1. Turn on BPO.
1. Turn off BPO.
1. Compare ACL field values.